tests/Element/TagBlockTest.php

```
@@ -136,6 +140,45 @@ public function testEndRemovesEmptyContextStackAfterBalancedRender(): void
+        $stackProperty = new ReflectionProperty(BaseBlock::class, 'stack');
+        $mainThreadProperty = new ReflectionProperty(BaseBlock::class, 'mainThread');
+
+        $stackProperty->setValue(null, null);
+        $mainThreadProperty->setValue(null, null);
+
+        TagBlock::tag()->begin();
+
+        $fiber = new Fiber(
+            static function (): void {
+                TagBlock::tag()->begin();
+                Fiber::suspend();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Fiber' in method 'testMaintainsIndependentBeginEndStackForFiber'. (undefined)

(StaticAccess)